code/pages/DMSDocumentCartCheckoutPage.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @property Text $ThanksMessage
+ * @property Int  $CartEmailRecipientID
+ *
+ * @method Member CartEmailRecipient
+ */
+class DMSDocumentCartCheckoutPage extends Page
+{
+    private static $db = array(
+        'ThanksMessage' => 'Text',
+    );
+
+    private static $has_one = array(
+        'CartEmailRecipient'     => 'Member'
+    );
+
+    private static $defaults = array(
+        'URLSegment'  => 'checkout',
+        'ShowInMenus' => false,
+    );
+
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+        $members = Member::get('Member');
+        $recipientDropDown = DropdownField::create(
+            'CartEmailRecipientID',
+            _t('DMSDocumentCartCheckoutPage.CART_RECIPIENT', 'Account to receive document print requests'),
+            $members->Map()->toArray()
```

Comment:
You could inline `Member::get()` here. The `'Member'` argument isn't required if you're calling it from a Member class